### ISSUE TITLE
last-release-sha seems to be ignored

### ISSUE THREAD

**Issue by lukeromanowicz** [google-github-actions/release-please-action#785]
### TL;DR

`last-release-sha` property seems to be ignored. `release-please-actions` is always trying to fetch the last release from API. 

### Expected behavior

When `last-release-sha` is set I expect it to be used as the cursor in git history starting from which the changelog is generated.  

### Observed behavior

I have the following releases: 0.1.0, 0.2.0, 0.2.1, 0.2.2. I'm trying to release version 0.3.0 which includes all the changes starting from 0.2.0. `cbf719c65` is the release commit of 0.2.0

I'm trying to modify the last release behavior with the following settings:

```
          last-release-sha: cbf719c65a1af9650998135c7f9e5a322179394e
          release-search-depth: 500
          commit-search-depth: 500
```
and it's always ignored. 

### Action YAML

```yaml
- uses: google-github-actions/release-please-action@v3.7.10
        if: env.IS_COMMIT_TAGGED == 'false'
        id: release-please
        with:
          release-type: node
          package-name: release-please-action
          release-as: ${{ env.RELEASE_VERSION }}
          default-branch: ${{ env.BRANCH_NAME }}
          last-release-sha: cbf719c65a1af9650998135c7f9e5a322179394e
          release-search-depth: 500
          commit-search-depth: 500
          token: ${{ secrets.BOT_GITHUB_TOKEN }}
          include-v-in-tag: false
```

### Log output

```text
Looking for latest release on branch: release/0.3.0 with prefix: release-please-action
❯ Fetching merge commits on branch release/0.3.0 with cursor: undefined
❯ Found latest release pull request: 311 version: 0.2.2
❯ Fetching releases with cursor undefined
⚠ Could not find releases.
❯ found 1 possible releases. [
  Version {
    major: 0,
    minor: 2,
    patch: 2,
    preRelease: undefined,
    build: undefined
  }
]
✔ Building releases
✔ Building strategies by path
❯ .: node
✔ Looking for latest release on branch: release/0.3.0 with prefix: release-please-action
❯ Fetching merge commits on branch release/0.3.0 with cursor: undefined
❯ Found latest release pull request: 311 version: 0.2.2
❯ Fetching releases with cursor undefined
⚠ Could not find releases.
❯ found 1 possible releases. [
  Version {
    major: 0,
    minor: 2,
    patch: 2,
    preRelease: undefined,
    build: undefined
  }
]
✔ Building pull requests
✔ Building strategies by path
❯ .: node
✔ Collecting release commit SHAs
❯ release search depth: 500
❯ Fetching releases with cursor undefined
⚠ Could not find releases.
⚠ Expected 1 releases, only found 0
⚠ Missing 1 paths: .
❯ looking for tagName: 0.2.2
❯ found: 0.2.2 9f047ecf870f45baa658847cc02400aa6c6c319f
❯ release for path: ., version: 0.2.2, sha: 9f047ecf870f45baa658847cc02400aa6c6c319f
✔ Collecting commits since all latest releases
❯ commit search depth: 500
```
```


### Additional information

_No response_

**Comment by chingor13** [comment]
Moving forward (v4), all advanced configuration options should be specified in the `release-please-config.json`.  https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
